darwin/main.m

```diff
@@ -237,3 +237,44 @@ void uiQueueMain(void (*f)(void *data), void *data)
 	// the signature of f matches dispatch_function_t
 	dispatch_async_f(dispatch_get_main_queue(), data, f);
 }
+
+@interface TimerDelegate : NSObject {
```

Comment:
Also you might as well make this change now as I'm going to be making this change throughout libui at some point in the near future anyway: rename the class `uiprivTimerDelegate`. I know it makes code a bit less nice to read, but unless I move everything (except the public API) to C++ it's either reserve the `uipriv` prefix for private functions that get exposed to a public namespace (which includes *all* Objective-C classes) or keep the crazy cmake static library hacks that don't work. :/

Reply:
The symbol stripping stuff on the GTK+ and macOS CMakeLists.txt and the resource file copying on Windows. Both have to do with quirks of static libraries, and the former is resolved by ensuring the namespace isn't polluted. (The latter is caused by a different issue, but the same problem applies.)

Reply:
Oh the `_handle_static` macro??

Definitely want to get rid of that, breaks the build on OpenBSD! (#229)

Reply:
All the more reason to fix this! =P